packages/darklang/languageTools/lsp-server/handleIncomingMessage.dark

```diff
@@ -4,55 +4,160 @@ module Darklang =
       /// This is how we handle incoming requests from the client.
       ///
       /// (this is the core of Darklang's LSP server)
```

Comment:
Could you please move this back to commenting the handleIncomingMessage fn in particular?